crates/fuel-core/src/service/config.rs

```diff
@@ -52,6 +52,7 @@ pub struct Config {
     /// When `true`:
     /// - Enables manual block production.
     /// - Enables debugger endpoint.
+    /// - Enables storage read replay for historical blocks.
```

Comment:
We can move this comment to `historical_execution`

Reply:
71a6a5607f